metaswitch/common/comm_monitor.py

```diff
@@ -52,16 +52,21 @@ def __init__(self, process, raise_identifier, clear_identifier):
 
     def set_alarm(self):
         self.alarmed = True
+        _log.warning("Raising alarm {}".format(self._raise_identifier))
         issue_alarm(self._process, self._raise_identifier)
 
     def clear_alarm(self):
         self.alarmed = False
+        _log.warning("Clearing alarm {}".format(self._clear_identifier))
         issue_alarm(self._process, self._clear_identifier)
 
     def update_alarm_state(self):
         now = monotonic_time()
         with self.mutex:
+            _log.debug("alarmed is {}, now is {}, next check time is {}, succeeded is {}, failed is {}"
```

Comment:
This log text could be more verbose, I don't think it's clear what's being reported